File: backend/parking-domain/src/main/java/com/chatchatabc/parking/infra/service/JedisService.java
package com.chatchatabc.parking.infra.service;

public interface JedisService {

    /**
     * Set key value to redis
     *
     * @param key   the key
     * @param value the value
     * @param ttl   the ttl
     */
    void set(String key, String value, Long ttl);

    /**
     * Get value from redis
     *
     * @param key the key
     * @return the value
     */
    String get(String key);

    /**
     * Delete key from redis
     *
     * @param key the key
     */
    void del(String key);
}

File: backend/parking-service/src/main/kotlin/com/chatchatabc/service/infra/service/JedisService.kt
package com.chatchatabc.service.infra.service

import org.springframework.stereotype.Service

@Service
interface JedisService {
    /**
     * Set key-value to Redis
     */
    fun set(key: String, value: String, ttl: Long?)

    /**
     * Get value from Redis
     */
    fun get(key: String): String?

    /**
     * Delete key from Redis
     */
    fun del(key: String)
}